utilities.tex

```diff
@@ -563,7 +596,37 @@
 For templates \tcode{greater}, \tcode{less}, \tcode{greater_equal}, and
 \tcode{less_equal}, the specializations for any pointer type yield a total order,
 even if the built-in operators \tcode{<}, \tcode{>}, \tcode{<=}, \tcode{>=}
-do not.
+do not. \ednote{The following sentence is taken from the proposed resolution of
+\href{https://cplusplus.github.io/LWG/lwg-active.html\#2450}{LWG \#2450}.}
+\newtxt{For template specializations \tcode{greater<void>}, \tcode{less<void>},
+\tcode{greater_equal<void>}, and \tcode{less_equal<void>}, if the call operator calls a
+built-in operator comparing pointers, the call operator yields a total order.}
+
+\newtxt{
+\pnum
+Given a relational operator \tcode{\textit{OP}}, an object \tcode{x} of type \tcode{X},
+and an object \tcode{y} of type \tcode{Y}, let \tcode{\textit{BUILTIN_PTR_CMP}(X, \textit{OP}, Y)}
```

Comment:
"object" won't work since `X`/`Y` may be lvalue reference types, this needs the silly "lvalue if `X` is an lvalue reference type or rvalue otherwise" dance.